docs/pages/includes/plugins/finish-event-handler-config.mdx

```diff
@@ -9,6 +9,12 @@ storage = "./storage"
 timeout = "10s"
 batch = 20
 namespace = "default"
+# The window size configures the duration of the time window for the event handler
+# to request events from Teleport. By default, this is set to 24 hours.
+# Reduce the window size if the events backend cannot manage the event volume 
```

Comment:
How do you know that the event backend cannot manage the event volume? An error? Timeout? Which event backend is subject to this limitation? Does this work with any backend?

Reply:
Currently we don't as Athena backend fails to without returning an error. Customers that need this need our assistance and

Reply:
Can you specify this solves an atherna-only issue? So regular users don't mess with this?

Reply:
It can also fix other backend issues like file storage where it opens all files prior to scanning them.
I think it's worth going as is